### PR TITLE
Upgrade to Java 8 update 171

### PR DESCRIPTION
**Pull request by pzakha**
Trunk has been already updated to version 171.
We need this to merge with trunk.

### TESTING
http://ops.jenkins.delphix.com/job/devops-gate/job/projects/job/dx4linux/job/appliance-build/job/master/job/pre-push/87/ (passed)

### CODE REVIEW

**Comment by prakashsurya** [comment]
It would be neat to attempt an upgrade with this change, and verify it works like we expect.

File: scripts/build-ancillary-repository.sh [hunk 1]
```diff
@@ -94,7 +94,7 @@ function build_delphix_java8_debs() {
 	local pkg_directory="$1"
 
 	local url="http://artifactory.delphix.com/artifactory"
-	local tarfile="jdk-8u131-linux-x64.tar.gz"
+	local tarfile="jdk-8u171-linux-x64.tar.gz"
 	local jcefile="jce_policy-8.zip"
 	local debfile="oracle-java8-jdk_8u131_amd64.deb"
```
Comment (prakashsurya) [inline scripts/build-ancillary-repository.sh]: you need to update this line, too.

File: scripts/build-ancillary-repository.sh [hunk 2]
```diff
@@ -94,7 +94,7 @@ function build_delphix_java8_debs() {
 	local pkg_directory="$1"
 
 	local url="http://artifactory.delphix.com/artifactory"
-	local tarfile="jdk-8u131-linux-x64.tar.gz"
+	local tarfile="jdk-8u171-linux-x64.tar.gz"
 	local jcefile="jce_policy-8.zip"
```
Comment (basil) [inline scripts/build-ancillary-repository.sh]: Java 8 Update 171 no longer requires a separate JCE Policy, as far as I know. (See [this review](http://reviews.delphix.com/r/40951/) for details, or ask Brandon.) I think this should therefore be deleted (along with the `--jce-policy` argument to `make-jpkg` below).

**Pull request merged**